src/Esprima/JavascriptParser.cs

```diff
@@ -4623,20 +4679,19 @@ private ArrayList<Decorator> ParseDecorators()
     {
         var decorators = new ArrayList<Decorator>();
 
-        while (Match("@"))
+        while (ConsumeMatch("@"))
```

Comment:
This way the leading `@` will be excluded from the range of the decorator. AST Explorer includes it, so I think this change should be reverted.